sormas-api/src/main/resources/captions.properties

```diff
@@ -145,6 +145,7 @@ ActivityAsCase.activityAsCaseType=Type of Activity
 ActivityAsCase.activityAsCaseTypeDetails=Type of Activity details
 ActivityAsCase.location=Location
 ActivityAsCase.typeOfPlace=Type of place
+ActivityAsCase.typeOfPlaceIfSG=Facility (IfSG)
```

Comment:
Not necessary; caption should be changed in Crowdin for the existing typeOfPlace property

Reply:
The distinction is necessary for the same reason as above.

Reply:
The string to be used is depending on the instance's country, not the language. So a instance in Germany with user language German should still show "Facility (IfSG)", whereas an instance in another country with user language German should show "Type of place".

Reply:
Absolutely true!